server/@types/bapv.d.ts

```diff
@@ -248,3 +248,14 @@ export type VisitsPageSlot = {
   adults: number
   children: number
 }
+
+export type Schedule = {
```

Comment:
I don't think this is needed? Duplicate of what is happening in `visitSchedulerApiTypes.ts` where it has:
```
export type SessionSchedule = components['schemas']['SessionScheduleDto']
```

Reply:
this is a relic from when I was doing the processing at the route level, will get this out tomorrow.